server/storeAppointment.php

```diff
@@ -90,7 +90,8 @@ function storeAppointment($data){
 				?,
 				(SELECT appointmentTimeId FROM AppointmentTime
 					WHERE DATE(scheduledTime) = ? 
-					AND time_format(TIME(scheduledTime), '%l:%i %p') = ?),
+					AND TIME_FORMAT(TIME(scheduledTime), '%l:%i %p') = ?
+					AND siteId = ?),
```

Comment:
@CollinsSpencer I also fixed what I think was a small bug here. I included the siteId in this query where you get the AppointmentTimeId for the time the user has inputted. Without the siteId, theoretically, if there was two AppointmentTimes that were happening on the same date at the same time, then this query would've returned 2 appointmentTimeIds and the whole query would've failed. Note that that is a valid case.

By including the siteId we should avoid that situation.